### PR TITLE
feat:  street manager notification email settings

### PR DESCRIPTION
**Pull request by DeannaSharma**
Note: Might be worth waiting until ticket DISRUPTION-597 (notification emails for new roadworks) is completed before merging this PR.

### CODE REVIEW

File: site/pages/api/admin/update-email-preference.api.ts [hunk 1]
```diff
@@ -16,8 +16,6 @@ const updateEmailPreference = async (req: NextApiRequest, res: NextApiResponse)
 
         if (!session) {
             throw new Error("No session found");
-        } else if (!session.isOrgAdmin) {
-            throw new Error("Invalid user accessing the page");
```
Comment (alainajaffer) [inline site/pages/api/admin/update-email-preference.api.ts:20]: Just wondering if the file path should be changes if we aren't checking if they are not an admin anymore. It is still api/admin...

Comment (alainajaffer) [reply site/pages/api/admin/update-email-preference.api.ts:20]: Perhaps we still want to check groups based on what email preference we are updating

Comment (alainajaffer) [inline site/pages/api/admin/update-email-preference.api.ts:20]: Emily said For any preferences e.g. email notifications or data sources, we only present these choices to organisation admin accounts currently - so I think we should still have this check

File: site/pages/account-settings.page.tsx [hunk 2]
```diff
@@ -222,34 +239,38 @@ const AccountSettings = ({
                               ]
                     }
                 />
-                {sessionWithOrg.isOrgAdmin ? (
```
Comment (alainajaffer) [inline site/pages/account-settings.page.tsx:225]: I think this check should still apply

File: site/pages/api/admin/update-email-preference.api.ts (hunk 1 above)
Comment (DeannaSharma) [reply site/pages/api/admin/update-email-preference.api.ts:20]: Checked with Emily and she's happy to leave code as is so we can set email preferences for street manager for org admins, publishers and staff.

**Review by alainajaffer — DISMISSED** [review]

**Review by alainajaffer — APPROVED** [review]

**Pull request merged**